lib/wordpress_uitlity.rb

```diff
@@ -0,0 +1,48 @@
+require 'rubypress'
+require 'mime/types'
+
+module WordpressUtility
+  class Post
+    FILENAME = 'cherry_PNG635.png'
+
+    def initialize
+      @wp = Rubypress::Client.new(host: "104.131.111.55",
```

Comment:
Pass all the hardcoded required configuration from outside. In our case pass from controller. In controller take all those parameters from environment variables. Think about the scenario when we would deploy this to heroku.